pytorch/training/buildspec-2-2-sm.yml

```diff
@@ -0,0 +1,66 @@
+account_id: &ACCOUNT_ID <set-$ACCOUNT_ID-in-environment>
+prod_account_id: &PROD_ACCOUNT_ID 763104351884
+region: &REGION <set-$REGION-in-environment>
+framework: &FRAMEWORK pytorch
+version: &VERSION 2.2.0
+short_version: &SHORT_VERSION "2.2"
+arch_type: x86
+autopatch_build: "True"
```

Comment:
note: if you want to avoid rebuilding the image from scratch, use the following:

1. comment out line 8 - `# autopatch_build: "True"`
2. uncomment line 47 and set to `build_tag_override: "beta:2.2.0-cpu-py310-ubuntu20.04-sagemaker-autopatch"`
3. uncomment line 64 and set to `build_tag_override: "beta:2.2.0-gpu-py310-cu121-ubuntu20.04-sagemaker-autopatch"`